CMakeLists.txt

```diff
@@ -37,4 +37,13 @@ add_executable(tmatrix ${tmatrix_SRC})
 # Specify the include directory.
 include_directories(include)
 
+# Specify how to compress the man page.
+configure_file(${PROJECT_SOURCE_DIR}/tmatrix.6 ${PROJECT_BINARY_DIR}/tmatrix.6)
+add_custom_command(
```

Comment:
Join the `OUPTUP` line with this line, use tabs instead of spaces for indenting the other commands (as it is already used in this file) and move the closing parenthesis to a new line.